### PR TITLE
Player 2375

### PR DESCRIPTION
**Pull request by atopilskii**
Delete old CastSampleApp v2 and rename new CastV3SampleApp to CastSampleApp

### CODE REVIEW

File: ChromecastSampleApp/app/build.gradle [hunk 1]
```diff
@@ -26,8 +26,8 @@ repositories {
 }
 
 task copyTask(type: Copy) {
-    from new File(vendorDir, 'Ooyala/OoyalaSDK-Android/OoyalaSDK.aar')
-    from new File(vendorDir, 'Ooyala/OoyalaCastSDK-Android/OoyalaCastSDK.aar')
+    from new File(vendorDir, 'Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
```
Comment (brol1dev) [inline ChromecastSampleApp/app/build.gradle]: we should be using .aar now, right?

File: ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastOptionsProvider.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.google.android.gms.cast.framework.OptionsProvider;
+import com.google.android.gms.cast.framework.SessionProvider;
+import com.google.android.gms.cast.framework.media.CastMediaOptions;
+import com.google.android.gms.cast.framework.media.NotificationOptions;
+
+import java.util.List;
+
+
+/**
+ * Don't delete this class. It is used to provide google cast options by manifest.
```
Comment (brol1dev) [inline ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastOptionsProvider.java]: You can safely remove the `Don't delete this class` comment :) Just explain what it's used for.

**Review by brol1dev — CHANGES_REQUESTED** [review]

File: ChromecastSampleApp/app/build.gradle (hunk 1 above)
Comment (atopilskii) [reply ChromecastSampleApp/app/build.gradle]: done

File: ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastOptionsProvider.java (hunk 2 above)
Comment (atopilskii) [reply ChromecastSampleApp/app/src/main/java/com/ooyala/sample/CastOptionsProvider.java]: Done

**Comment by brol1dev** [comment]
Can you update this PR's branch with the latest changes from `dev`?

And there are also merge conflicts.

**Comment by atopilskii** [comment]
done

**Review by akshaydch11 — APPROVED** [review]

**Pull request merged**